packages/gfbgraph.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gfbgraph < Package
+class Gfbgraph < Autotools
   description 'GLib/GObject wrapper for the Facebook Graph API'
   homepage 'https://wiki.gnome.org/Projects/GFBGraph'
   version '0.2.4'
```

Comment:
Update to [0.2.5](https://gitlab.gnome.org/GNOME/libgfbgraph/-/tags/v_0_2_5)?

Reply:
No package 'rest-0.7' found   ...